src/test/java/redis/clients/jedis/modules/search/AggregationWithParamsTest.java

```diff
@@ -0,0 +1,297 @@
+package redis.clients.jedis.modules.search;
+
+import static org.junit.Assert.*;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import redis.clients.jedis.exceptions.JedisDataException;
+import redis.clients.jedis.search.*;
+import redis.clients.jedis.search.aggr.*;
+import redis.clients.jedis.modules.RedisModuleCommandsTestBase;
+
+public class AggregationWithParamsTest extends RedisModuleCommandsTestBase {
```

Comment:
I'm assuming the commented-out tests are placeholders from JRediSearch?

Reply:
@bsbodden Kinda yes but these are also available in src/test/java/redis/clients/jedis/modules/search/AggregationBuilderTest.java. Things are still in progress :smiley: